### ISSUE TITLE
Error from `postinstall` with version 5.5.0 from NPM

### ISSUE THREAD

**Issue by surilindur** [LinkedSoftwareDependencies/Components.js#135]
#### Issue type:

- :bug: Bug 

____
#### Description:

When version 5.5.0 is installed from NPM, it will try to do `yarn build` because of the `postinstall` script, which does not seem necessary because the package has already been built before publishing to NPM. The script also throws an error because the build script uses something called `scopy`.

The `postinstall` script was added in commit [680c7cb](https://github.com/LinkedSoftwareDependencies/Components.js/commit/680c7cba0479879e334976680c16531403b4feb6), but the `scopy` command has been there longer.

I do not know what `scopy` is, but the script looks like it could be replaced with `cp`. All the Google results I get for `scopy` are about some kind of signal analysis toolkit.

Could the `postinstall` script be either removed as unnecessary, the `scopy` call replaced with `cp`, or both? If needed, I can make either or both changes, but I wanted to ask for feedback before making a PR.

Thank you in advance for your time!

____
#### Environment:

* Components.js 5.5.0
* Node.js 21.5.0
* Yarn 3.6.4 or 4.0.2 (tested both)
* Linux/UNIX

#### Crash log:

Yarn output (relevant part):

```
➤ YN0009: │ componentsjs@npm:5.5.0 couldn't be built successfully (exit code 127, logs can be found here: /tmp/xfs-9407fa40/build.log)
```

Contents of `/tmp/xfs-9407fa40/build.log`:

```
# This file contains the result of Yarn building a package (componentsjs@npm:5.5.0)
# Script name: postinstall

command not found: scopy
```

**Comment by jaxoncreed** [comment]
Encountering this when installing the new version too:

NPM: 8.19.3
Node: v18.13.0

**Comment by rubensworks** [comment]
Hmm, this was introduced in https://github.com/LinkedSoftwareDependencies/Components.js/commit/680c7cba0479879e334976680c16531403b4feb6#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519
My bad, didn't look close enough at the modified scripts...

Released a fix as 5.5.1.
